build/images/deps/suricata-version

```diff
@@ -1 +1 @@
-6.0
+7.0
```

Comment:
is the bumping up related to L7Visibility? If not, it should be done separately.